### PR TITLE
[master] [KOGITO-1359] Setup jobs-service to data-index feature test

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-1359

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
I will cherry-pick after in 0.9.x

File: test/features/install_jobs_service.feature [hunk 1]
```diff
@@ -43,4 +43,27 @@ Feature: Install Kogito Jobs Service
     And Scale Kogito Jobs Service to 0 pods within 2 minutes
     And Scale Kogito Jobs Service to 1 pods within 2 minutes
 
-    Then HTTP GET request on service "kogito-jobs-service" with path "jobs/1" is successful within 1 minutes
\ No newline at end of file
+    Then HTTP GET request on service "kogito-jobs-service" with path "jobs/1" is successful within 1 minutes
+
+#####
```
Comment (spolti) [inline test/features/install_jobs_service.feature:48]: leftover?

Comment (radtriste) [reply test/features/install_jobs_service.feature:48]: nope, this is made to have a clear view of separation between the tests :)

File: test/steps/graphql.go [hunk 2]
```diff
@@ -57,3 +58,25 @@ func (data *Data) graphqlRequestOnDataIndexReturnsProcessInstancesProcessNameWit
...
+	framework.GetLogger(data.Namespace).Debugf("graphqlRequestOnDataIndexReturnsJobsIDWithinMinutes with service %s, path %s, query %s and timeout %d", serviceName, path, query, timeoutInMin)
+	routeURI, err := framework.WaitAndRetrieveRouteURI(data.Namespace, serviceName)
+	if err != nil {
+		return err
+	}
+	response := GraphqlDataIndexJobsQueryResponse{}
+	return framework.WaitForSuccessfulGraphQLRequest(data.Namespace, routeURI, path, query, timeoutInMin, &response, func(response interface{}) (bool, error) {
+		resp := response.(*GraphqlDataIndexJobsQueryResponse)
+		for _, processInstance := range resp.Jobs {
+			if processInstance.ID == id {
```
Comment (MarianMacik) [inline test/steps/graphql.go:76]: Isn't this ID of a job in general?

Comment (radtriste) [reply test/steps/graphql.go:76]: this is the job, I did not change the variable name in copy/paste...

**Comment by radtriste** [comment]
@MarianMacik are you good with latest change ?

**Pull request merged**